bedrock/mozorg/redirects.py

```diff
@@ -629,4 +629,7 @@ def decider(request, **kwargs):
 
     # Bug 1313023
     redirect(r'^story/?$', 'https://donate.mozilla.org/?source=story_redirect'),
+
+    # Bug 682619
+    redirect(r'^support/thunderbird/?$', 'https://support.mozilla.org/en-US/products/thunderbird'),
```

Comment:
This is failing due to the existing redirect up on line 594. Moving this new redirect immediately above the existing one should fix things:

```
# bug 1236910, 682619
redirect(r'^support/thunderbird/?$', 'https://support.mozilla.org/products/thunderbird'),
redirect(r'^support(/.*)?$', 'https://support.mozilla.org/'),
```

Note: I also removed `/en-US/` from the redirect there. This will allow SUMO to choose the best language for the user based on their accept language header, rather than always redirecting to the English page.

Reading the bug comments, it might also be worth redirecting `/support/firefox/` to `https://support.mozilla.org/products/firefox` while we're here.